03-11-2018/HybernateExample/src/main/java/com/hibernate/daoImpl/DatabaseConnectionDaoImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.hibernate.daoImpl;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.service.ServiceRegistry;
+
+import com.hibernate.dao.DatabaseConnectionDao;
+import com.hibernate.model.UserDetails;
+
+public class DatabaseConnectionDaoImpl implements DatabaseConnectionDao
+{
+	public void createTable()
+	{
+		try
+		{
+			Configuration config=new Configuration();
```

Comment:
Create sessionfactory in separate method . dont use ServiceRegistry in every method